### PR TITLE
Return active running jobs on the master

### PR DESCRIPTION
**Pull request by austinpapp**
### What does this PR do?
Return all active jobs running on a master. This is follow up work to #51629

Secondary updates:
* `salt/runner.py::_run_runner` was always returning `{retcode: 70}`. removed the `else` logic. This logic was merged in here https://github.com/saltstack/salt/pull/48447
* fixed some runner tests

### Tests written?

Yes

### Commits signed with GPG?

Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Review by twangboy — APPROVED** [review]

**Comment by twangboy** [comment]
I have a similar PR here: https://github.com/saltstack/salt/pull/52158

**Comment by cachedout** [comment]
@austinpapp Some test trouble here I'm afraid.

**Comment by austinpapp** [comment]
@cachedout yea. Will fix when I have a few free cycles.

File: salt/runners/jobs.py [hunk 1]
```diff
@@ -31,6 +32,20 @@
...
+    Return the actively executing runners for the master
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt-run jobs.master
+    '''
+    jobs = salt.utils.master.get_running_jobs(__opts__)
+    return jobs
```
Comment (cachedout) [inline salt/runners/jobs.py]: Nitpick: collapse into one line. :)

File: tests/integration/runners/test_jobs.py [hunk 2]
```diff
@@ -74,7 +86,7 @@ def test_target_info(self):
         ret = self.run_run_plus('jobs.list_jobs')
         for item in ret['return'].values():
             if item['Function'] == 'test.echo' and \
-                item['Arguments'][0] == 'target_info_test':
+               item['Arguments'][0] == 'target_info_test':
```
Comment (cachedout) [inline tests/integration/runners/test_jobs.py]: Needs one more space to align with line below.

File: salt/runners/jobs.py (hunk 1 above)
Comment (austinpapp) [reply salt/runners/jobs.py]: meh. but because its mike, i'll do it for you :D

Comment (cachedout) [reply salt/runners/jobs.py]: VICTORY! 🎉

**Comment by austinpapp** [comment]
these tests 🤦‍♂️ ...

**Review by cachedout — APPROVED** [review]

**Comment by Ch3LL** [comment]
I ported this PR to the master branch here: https://github.com/saltstack/salt/pull/56440 

We have changed to using the master branch instead of develop going forward. You can see further details about this change here: https://github.com/saltstack/salt-enhancement-proposals/pull/20

**Pull request closed without merging**